### ISSUE TITLE
I have news on MyBook (JMICRON board)

### ISSUE THREAD

**Issue by Zibri** [themaddoctor/linux-mybook-tools#88]
If you have an original MY BOOK board with JMICRON JMS579 on it, I have good news.
I am reversing (hardware and software) that board and found many nice things like:

1) the flash can be fully dumped without even needing a flash programmer.
2) any firmware can be flashed without even opening the device.

I am willing to help on this but I first need a favour:
if any of you have this board, please contact me (zibri AT zibri.org).

**Comment by themaddoctor** [comment]
Are you saying that you can dump the EEPROM contents through the USB connection?
Would your method be generic enough to use with a JMS538S?

**Comment by Zibri** [comment]
Yes I can do that on JMS578/JMS579.I never saw JMS538S so I don’t know.  From: themaddoctorSent: Sunday, March 5, 2023 03:34To: themaddoctor/linux-mybook-toolsCc: Zibri; AuthorSubject: Re: [themaddoctor/linux-mybook-tools] I have news on MyBook (JMICRON board) (Issue #88) Are you saying that you can dump the EEPROM contents through the USB connection?Would your method be generic enough to use with a JMS538S?—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>

**Comment by themaddoctor** [comment]
If you share, I can try it.

**Comment by Zibri** [comment]
we need to "meet" on some chat (fb/twitter) I have to guide you step by step.
But you need to have a board that has not be flashed. that has the original WD MY BOOK firmware.

**Comment by themaddoctor** [comment]
I have such a board. Would it matter if it has a drive attached? Whether it is the original drive?

**Comment by Zibri** [comment]
> I have such a board. Would it matter if it has a drive attached? Whether it is the original drive?

No, it matters only that it has the original untouched firmware and configuration. (the full flash)

**Comment by ojfd** [comment]
Did you know, guys, that there is a whole web page dedicated to JMicron flashers, firmwares and tools?
https://www.usbdev.ru/files/jmicron/
Use translator and have fun.
